package.json

```diff
@@ -103,6 +103,7 @@
     "browserify": "^16.2.3",
     "bundlesize": "^0.17.2",
     "chalk": "^2.4.1",
+    "chrome-devtools-frontend": "^1.0.708769",
```

Comment:
fyi this hasn't been published in a month

Reply:
thanks for the heads up. fixed.

1.0.723630 is today's ToT

Reply:
fyi i am on `1.0.723630` rn just didnt' update the package.json semever